src/elm/Main.elm

```diff
@@ -4,7 +4,6 @@ import App.Model exposing (Model)
 import App.Router exposing (..)
 import App.Update exposing (init, update, Msg)
 import App.View exposing (view)
-import Html.App as Html
```

Comment:
@amitaibu `elm-format` recommended removing this as it's not being used but seems weird to me to remove it, Is it ok ?